app/display/model/src/main/java/org/csstudio/display/builder/model/util/ModelResourceUtil.java

```diff
@@ -421,7 +431,7 @@ public static InputStream openResourceStream(final String resource_name) throws
 //            final long milli = Math.round(1000 + Math.random()*4000);
 //            Thread.sleep(milli);
 //        }
-        if (resource_name.startsWith("http") || resource_name.startsWith("file:/"))
+        if (resource_name.startsWith("http"))

```

Comment:
https://github.com/ControlSystemStudio/phoebus/pull/2967 just added the "file:/" check here, might be a problem to remove that

Reply:
I'll reroll that